include/aspect/material_model/interface.h

```diff
@@ -217,6 +217,16 @@ namespace aspect
                           const bool use_strain_rates = true);
 
       /**
+       * Copy Constructor.
+       */
+      MaterialModelInputs(const MaterialModelInputs<dim> &material);
+
+      /**
+       * Assignment operator.
+       */
+      MaterialModelInputs &operator=(const MaterialModelInputs &material);
```

Comment:
Of course I can just revert this. But I was just thinking of @MFraters. Those tests where the assignment was used were all related to the Newton solver, and I was thinking he might have a number of his own plugins and test cases that might use a similar logic.